third_party/2and3/click/core.pyi

```diff
@@ -228,6 +228,9 @@ class Command(BaseCommand):
     def make_parser(self, ctx: Context) -> OptionParser:
         ...
 
+    def get_short_help_str(self, limit: int) -> str:
```

Comment:
limit has a default: https://github.com/pallets/click/blob/master/click/core.py#L887

should add `= ...`